compass/landice/tests/thwaites/decomposition_test/__init__.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, test_group):
 
         Parameters
         ----------
-        test_group : compass.landice.tests.thwaites
+        test_group : compass.landice.tests.thwaites.Thwaites
```

Comment:
@matthewhoffman, pycharm pointed out this type issue, which I took the liberty of fixing